### PR TITLE
[ Packaging for AGE ] added packaging scripts : control,rules,format

### PR DESCRIPTION
**Pull request by sarthak-19**
Clean the directory and added scripts for debian package build. 
The deb file can be build using `dpkg-buildpackage -b `

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+#override_dh_auto_test:
+#	# nothing to do here, see debian/tests/* instead
+#
+#override_dh_auto_install:
+#	+pg_buildext install build-%v postgresql-%v-age
+#
+#override_dh_auto_clean:
+#	+pg_buildext clean build-%v
+#
```
Comment (uhayat) [inline debian/rules:23]: Remove the dead code L#8 till L#23

File: debian/gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+include: https://salsa.debian.org/postgresql/postgresql-common/raw/master/gitlab/gitlab-ci.yml
```
Comment (uhayat) [inline debian/gitlab-ci.yml:1]: No need for this file and its incorrect

File: debian/control [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+# under the License.
+
+
+Source: age
+Section: database
+Priority: optional
+Maintainer: Debian PostgreSQL Maintainers <team+postgresql@tracker.debian.org>
+Uploaders:
+  Sarthak Jaiswal <sarthak@debian.org>,
+  Ahmar Zaidi <ahmar@debian.org>,
```
Comment (uhayat) [inline debian/control:25]: Use AGE maintainer email address from L#22 till L#25

File: debian/control [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Build-Depends:
+ debhelper-compat (= 13),
+ postgresql-11 | postgresql-12,
```
Comment (uhayat) [inline debian/control:28]: Target only PG12

File: debian/control [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+ build-essential,
+ libreadline-dev,
+ zlib1g-dev,
+ flex,
+ bison,
+Standards-Version: 4.5.1
+Rules-Requires-Root: no
+Homepage: https://age.apache.org
+Vcs-Browser: https://salsa.debian.org/postgresql/age
+Vcs-Git: https://salsa.debian.org/postgresql/age.git
```
Comment (uhayat) [inline debian/control:38]: L#37,38 not correct and should removed or fixed

**Review by uhayat — CHANGES_REQUESTED** [review]

**Comment by sarthak-19** [comment]
Made the suggested changes. Request for review and merge

**Review by uhayat — APPROVED** [review]
Looks good

**Pull request merged**